src/js/themes/base.js

```diff
@@ -148,6 +148,12 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         },
         large: {}, // anything above 'medium'
       },
+      // Breakpoints used at SSR in order to guess a responsive breakpoint
+      ssrBreakpoints: {
```

Comment:
I suspect most people reading the code will not know what "ssr" means. And, we tend to prefer spelling out acronyms to help with this. But, what if we just called it `deviceBreakpoints`?

Also, can we leverage the existing `global.breakpoints` directly and just have these map to the existing names? For instance:
```
deviceBreakpoints: {
  phone: 'small',
  tablet: 'medium',
}
```

Reply:
agreed for the naming! its for sure better. i am not sure i understood though what you suggest for the breakpoints. 
do you mean something like
```
 {
  breakpoints: {
    small: {
      value: 600,
    },
    medium: {
      value: 800,
    }
  },
  deviceBreakpoints: {
    phone: 'small',
    tablet: 'medium',
  },
}
```

if it is that, i also thought about it but decided against because a user who changes the breakpoints might not necessarily change the name of the device ones and then SSR would not work as intended. but i might be thinking of a weird edge case so i wouldn't mind changing to that!

Reply:
i see yes! done :)